docs/guide/configuration/zigbee-network.md

```diff
@@ -43,6 +45,26 @@ Set `network_key: GENERATE` to let Zigbee2MQTT generate a new random key on the
 channel](../../advanced/zigbee/02_improve_network_range_and_stability.html#reduce-wi-fi-interference-by-changing-the-zigbee-channel.md)
 :::
 
+### Changing the Zigbee channel
+
+Changing the channel of an existing Zigbee network is supported. In Zigbee, this is done by broadcasting a network update indicating the channel change. However, there is a caveat: devices that are asleep during the broadcast will not switch to the new channel and will thus become unreachable. These are typically battery-powered end devices, and the only way to move them to the new channel is by re-pairing them.
```

Comment:
That's not quite what I've observed (from my own tests and from users feedback). Sleepy switch just fine in general (after they wake up, they notice they are no longer connected and trigger a scan to rejoin).
The issue with devices not following seem to be more about not following spec than their power type.

The caveat to be noted would be that sleepy may only rejoin the next time they wake up, which, for devices like basic motion sensors, or door sensors, may not be until they are triggered again.
_Advice would be to go around the house a few minutes after the channel change is effective, to wake everything up._ 😁

Reply:
Good to know, updated the docs.